javascript/schema/src/codebook/ColumnError.ts

```diff
@@ -0,0 +1,8 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+export interface ColumnError {
```

Comment:
It could be useful to add a list of cell indexes to this so we have the option of identifying the exact rows that have failures. We may not want to do that every time for performance reasons (see comment on validateTable), but when we do we could use it to highlight each cell that failed when viewing the table.

Reply:
Hmm. An alternative idea here that may have better performance would be to just pass the validation results to the column when rendering (i.e., in ArqueroDetailsList). Then if the column has any rules violated we could test the cells as we render them. It might be more expensive in the render loop, but we'd also only need to test the visible cells with lazy loading.